src/Map/Controller/DragIcon.java

```diff
@@ -139,6 +143,13 @@ public static MapNode constructMapNodeFromType(NodeType type)
 				break;
 			}
 
+			case Stairs:
```

Comment:
Hey, didn't we agree to change this to the enum type? Does that mean we can remove "set is ___"